daemon/container.go

```diff
@@ -154,7 +155,7 @@ func (daemon *Daemon) newContainer(name string, operatingSystem string, config *
 	base.Name = name
 	base.Driver = daemon.imageService.StorageDriver()
 	base.OS = operatingSystem
-	return base, err
+	return base, nil
```

Comment:
Probably this can be a separate commit

Reply:
Oh! I wanted to, but looks like I didn't do so. Let me put it separate.

Reply:
done 👍